Kp_x: 0.5
Ki_x: 0.0
Kp_y: 0.5
Ki_y: 0.0
Kp_z: 0.5
Ki_z: 0.0